sass/_a3-home-middle.scss

```diff
@@ -0,0 +1,54 @@
+//home middle styling
+//this is the "get it for free" title, paragraph, and google play and ios app store buttons.
+.a3-home-middle {
+  @extend %section-layouts;
+  background-color: $background-C;
+  & .slant {
+    @extend %svg;
+    fill: $background-A;
+    background-color: transparent;
+  }
+}
+.home-arrow {
+	margin-left: 8em;
+	margin-bottom: 0;
+}
+.home-middle-p {
+	@extend %small-font;
+	margin: 0;
+	padding-bottom: 3em;
+}
+.home-google-ios li{
+	display: inline-block;
+}
+.home-google-ios {
+	a {
+		color: $white;
```

Comment:
You should be consistent with your indentation here; you've got two spaces up top, then tabs lower down. It's important to decide on the code formatting as a team and then run with it.